src/WidgetBase.ts

```diff
@@ -242,15 +257,14 @@ export class WidgetBase<P extends WidgetProperties> extends Evented implements W
 	}
 
 	public setProperties(properties: P): void {
-		const diffPropertyResults: { [index: string]: PropertyChangeRecord } = {};
+		const diffPropertyResults: { [index: string]: any } = {};
```

Comment:
That's the weird thing. I don't think it was ever actually a `PropertyChangedRecord` 😄 (https://github.com/dojo/widget-core/blob/master/src/WidgetBase.ts#L263). It looks like, at some point, instead of storing property diff results in a single variable it was separated out into `diffPropertyResults` for the values, and `diffPropertyChangedKeys` for the keys.